docs/deploy.md

```diff
@@ -135,3 +135,54 @@ frontend:
 | /<*>.md        | /<*>.md        | 200 (Rewrite) |
 | /<*>.png       | /<*>.png       | 200 (Rewrite) |
 | /<*>           | /index.html    | 200 (Rewrite) |        
+
+
+## Docker
+
+>There is a simple demo for building docsify in docker.
+
+?> tips: you can get all the files for this demo from [here](https://github.com/Koooooo-7/docsify-docker).
+
+- Preparing Docsify initial files  
+  You need prepare the initial files (generated by run `docsify init ./docs`) instead of making in container.
```

Comment:
Users can also manually create the `index.html` per the instructions in the [Quickstart](https://docsify-preview-csiiei9ph.vercel.app/#/quickstart?id=manual-initialization) section of the docs. It would be nice to point this out for those that don't want to use `docsify-cli`.

Reply:
Got it.